Formula/halide.rb

```diff
@@ -2,8 +2,8 @@ class Halide < Formula
   desc "Language for fast, portable data-parallel computation"
   homepage "https://halide-lang.org"
   # TODO: Check if we can use unversioned `llvm` at version bump.
```

Comment:
Did we try this?

Reply:
Verified locally (Monterey x86_64) that it builds and tests successfully with unversioned `llvm`.